imports/plugins/included/marketplace/server/publications.js

```diff
@@ -73,3 +73,46 @@ Meteor.publish("SellerShops", function (shopIds) {
 
   this.ready();
 });
+
+/**
+ * MarketplaceShopPackages
+ * @summary function to publish packages available for a marketplace shop
+ * @param {string} shopID
+ * @return {object | boolean }
+ */
+Meteor.publish("MarketplaceShopPackages", function (shopId) {
+  check(shopId, String);
+
+  if (!Reaction.hasPermission("admin", this.userId, Reaction.getPrimaryShopId())) {
```

Comment:
@impactmass should this be `shopId` instead of `Reaction.getPrimaryShopId()`, and if not, could you explain why? (I'm not super clear on all of the roles... i.e., does a Marketplace Shop Owner have an admin role in the Primary Shop?)
thanks!